editor/index.js

```diff
@@ -22,6 +23,8 @@ import { initializeMetaBoxState } from './store/actions';
 export * from './components';
 import store from './store'; // Registers the state tree
 
+registerCoreBlocks();
```

Comment:
Maybe we should move this inside the "initialize" function since we do not support multiple instances anyway.

Reply:
Happy to move it there. I wasn't sure what is the best place to put it :)

In the long run, we could mirror what WP core uses and introduce some global hooks like `init` to take advantage of the `priority` param and be sure that `registerCoreBlocks` get called last after all other hooks coming from the plugins are registered.